docs/conf.py

```diff
@@ -22,10 +22,13 @@
 # extensions
 extensions = [
     "myst_nb",
-    # "autodoc_traits",  # TODO investigate if can help streamline documentation writing
     "sphinx.ext.autosummary",
```

Comment:
Should these be sorted alphabetically? Can't remember if order matters for sphinx extensions and a quick search didn't say it mattered...  This is the only thing I've noticed, everything looks like good improvements.

Reply:
yep order matters